Childrens-Social-Care-CPD/Views/Shared/_VideoResource.cshtml

```diff
@@ -8,9 +8,34 @@
 
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-three-quarters">
-        <video id="videoPlayer" preload="none">
-            <source src="@Model.Video.File.Url">
-        </video>
+        @if(!string.IsNullOrEmpty(Model.EmbededTitle) && !string.IsNullOrEmpty(Model.EmbededSourceUrl))
+        {
+            <h2>
+                @Model.EmbededTitle
+            </h2>
+            if (@Model.EmbededSourceUrl.IndexOf("vimeo") > -1)
+            {
+                <iframe width="560" height="315"
+                        src="@Model.EmbededSourceUrl"
+                        title="@Model.EmbededTitle"
+                        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                        allowfullscreen>
+                </iframe>
+            }
+            else
+            {
+                <video name="media" src="@Model.EmbededSourceUrl">
+                    <track label="English" kind="captions" srclang="en" default>
```

Comment:
The track element has no src attribute so not sure what benefit this is?

Reply:
Sonarqube issue